src/lib/ReactDataTable/TableRows.tsx

```diff
@@ -6,14 +6,17 @@ import { CSS } from "@dnd-kit/utilities";
 interface TableRowProps<TData> {
   row: Row<TData>;
   enableRowSelection?: boolean | ((row: Row<TData>) => boolean);
+  enableExpanding?: boolean | ((row: Row<TData>) => boolean);
   rowStyle?: CSSProperties;
   setNodeRef?: (node: HTMLElement | null) => void;
 }
 
 const InternalTableRow = <TData,>(props: TableRowProps<TData>) => {
   const { row, rowStyle, setNodeRef, enableRowSelection = false } = props;
+  //const { row, rowStyle, setNodeRef, enableRowSelection = false, enableExpanding = false } = props;
```

Comment:
@skuengneo I think this commented out code is not neeeded :)

Reply:
@manni497 You are right :D